src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Automation/Peers/DateTimeAutomationPeer.cs

```diff
@@ -30,10 +30,7 @@ public sealed class DateTimeAutomationPeer : AutomationPeer, IGridItemProvider,
         internal DateTimeAutomationPeer(DateTime date, Calendar owningCalendar, CalendarMode buttonMode)
             : base()
         {
-            if (date == null)
-            {
-                throw new ArgumentNullException("date");
-            }
+            // DateTime is a non-nullable value (a struct) type and cannot be null
```

Comment:
Remove this comment.  It's helpful for this review, but confusing to readers of the (changed) code, as it refers to code that's not there.